growth/too/tasks/tiles.py

```diff
@@ -43,7 +44,12 @@ def params_struct(dateobs, tobs=None, filt=['r'], exposuretimes=[60.0],
     growthpath = os.path.dirname(growth.__file__)
     config_directory = os.path.join(growthpath, 'too', 'config')
     tiling_directory = os.path.join(growthpath, 'too', 'tiling')
-    catalog_directory = os.path.join(growthpath, 'too', 'catalog')
+
+    catalog = os.path.join(app.instance_path, 'CLU.hdf5')
```

Comment:
Again, please use `app.open_instance_resource`, `pkg_resources.resource_stream`, and `try`/`except` here.